aixplain/modules/finetune/hyperparameters.py

```diff
@@ -56,6 +65,12 @@ def __post_init__(self):
         if self.epochs > EPOCHS_MAX_VALUE:
             raise ValueError(f"epochs must be one less than {EPOCHS_MAX_VALUE}")
 
+        if self.train_batch_size not in BATCH_SIZE_VALUES:
+            raise ValueError(f"train_batch_size must be one of the following values: {BATCH_SIZE_VALUES}")
+
+        if self.eval_batch_size not in BATCH_SIZE_VALUES:
+            raise ValueError(f"eval_batch_size must be one of the following values: {BATCH_SIZE_VALUES}")
+
         if self.max_seq_length > MAX_SEQ_LENGTH_MAX_VALUE:
```

Comment:
less or equal

Reply:
For batch size, I want only to receive values from the list BATCH_SIZE_VALUES, that's why we have `not in` there.